code/modules/spells/spell_types/lesserlichdom.dm

```diff
@@ -0,0 +1,151 @@
+/obj/effect/proc_holder/spell/targeted/lesserlichdom
+	name = "Lesser Bind Soul"
+	desc = "A weak version of the dark necromantic pact that can forever bind your soul to an \
+	item of your choosing. So long as both your body and the item remain \
+	intact and on the same plane you can revive from death for a limited number of times, though the time \
+	between reincarnations grows massively with use, along with the weakness \
+	that the new skeleton body will experience upon 'birth'. Note that \
+	becoming a lesser lich destroys all internal organs except the brain."
+	school = "necromancy"
+	charge_max = 10
+	clothes_req = FALSE
+	centcom_cancast = FALSE
+	invocation = "MINUS POTENS NECREM IMORTIUM!"
+	invocation_type = "shout"
+	range = -1
+	level_max = 0 //cannot be improved
+	cooldown_min = 10
+	include_user = TRUE
+
+	action_icon = 'icons/mob/actions/actions_spells.dmi'
+	action_icon_state = "skeleton"
+
+/obj/effect/proc_holder/spell/targeted/lesserlichdom/cast(list/targets,mob/user = usr)
+	for(var/mob/M in targets)
+		var/list/hand_items = list()
+		if(iscarbon(M))
+			hand_items = list(M.get_active_held_item(),M.get_inactive_held_item())
+		if(!hand_items.len)
+			to_chat(M, "<span class='caution'>You must hold an item you wish to make your phylactery...</span>")
+			return
+		if(!M.mind.hasSoul)
+			to_chat(user, "<span class='caution'>You do not possess a soul.</span>")
+			return
+
+		var/obj/item/marked_item
+
+		for(var/obj/item/item in hand_items)
+			// I ensouled the nuke disk once. But it's probably a really
+			// mean tactic, so probably should discourage it.
+			if((item.item_flags & ABSTRACT) || HAS_TRAIT(item, TRAIT_NODROP) || SEND_SIGNAL(item, COMSIG_ITEM_IMBUE_SOUL, user))
```

Comment:
does this block soul shards if they already have a soul?

Reply:
If the original lichdom spell does, this should as well, but I dont believe it does

Reply:
It seems neither the original wizard spell, or this weaker version blocks soul shards

Reply:
Do soul shards return false to COMSIG IMBUE SOUL if they already have one?

Reply:
Nope, seems they just add a soul to contents, and if contents contains anything then they're used, or something like that

Reply:
Alright, well, unrelated change. Just leave it for now - or if you want you can register the IMBUE_SOUL comsig on soul shards and refactor them to deny it if they have a soul.